modules/gateway/nodes.go

```diff
@@ -133,9 +144,10 @@ func (g *Gateway) requestNodes(conn modules.PeerConn) error {
 		return err
 	}
 
+	const inbound = true
```

Comment:
Curious to set this as a const. I think it's the right thing to do, but I don't think it's a pattern we've used anywhere else.